src/bindings/jna/libelektra4j/src/test/java/org/libelektra/README.md

```diff
@@ -0,0 +1,22 @@
+Unit tests can be run after importing jUnit, JNA and the libelektra4j java library into a project (eclipse, netbeans, intelliJ, ...). Alternatively, you can use maven to both build the project or just run the unit tests.
+
+Tested library versions are:
+	JNA: 4.2.2
+	jUnit: 4.12 [jUnit 3 is not supported]
+	hamcrest-core: 1.13 (required by newer jUnit versions)
+	Both libraries should work on version 4.0 too, though.
+
```

Comment:
I think we could add a few lines about the tested Java version. I've used openJDK 1.8.0 x86_64